conda/meta.yaml

```diff
@@ -1,19 +1,12 @@
 package:
   name: conda
-  version: 3.8.4
+  version:  '{{ environ["GIT_DESCRIBE_TAG"] }}.post{{ environ["GIT_DESCRIBE_NUMBER"] }}+{{ environ["GIT_DESCRIBE_HASH"] }}'
 
 source:
-  git_url: git@github.com:conda/conda.git
-  git_tag: 3.8.4
+  git_url: https://github.com/conda/conda.git
 
 build:
-  # These are present when the new environment is created
-  # so we have to exempt them from the list of initial files
-  # for conda-build to realize they should be included.
-  always_include_files:
-    - bin/activate
-    - bin/deactivate
-    - bin/conda
```

Comment:
In https://github.com/pelson/raspberrypi-conda-recipes/blob/master/recipes/conda/meta.yaml I had to include conda (but activate and deactivate are coming from conda-env) so that makes sense. I think you just need the single "conda" item.